scripts/byron-to-alonzo/mkfiles.sh

```diff
@@ -583,23 +583,31 @@ echo "$ROOT/run/all.sh"
 echo
 echo "In order to do the protocol updates, proceed as follows:"
 echo
-echo "  0. wait for the nodes to start producing blocks"
-echo "  1. invoke ./scripts/byron-to-alonzo/update-1.sh"
+echo "  0. invoke ./scripts/byron-to-alonzo/mkfiles.sh"
+echo "  1. wait for the nodes to start producing blocks"
+echo "  2. invoke ./scripts/byron-to-alonzo/update-1.sh"
 echo "     wait for the next epoch for the update to take effect"
 echo
-echo "  2. invoke ./scripts/byron-to-alonzo/update-2.sh"
-echo "  3. restart the nodes"
+echo "  3. invoke ./scripts/byron-to-alonzo/update-2.sh"
+echo "  4. restart the nodes"
 echo "     wait for the next epoch for the update to take effect"
+echo "     you should be in the Shelley era if the update was successful"
 echo
-echo "  4. invoke ./scripts/byron-to-alonzo/update-3.sh <N>"
+echo "  5. invoke ./scripts/byron-to-alonzo/update-3.sh <N>"
 echo "     Here, <N> the current epoch (2 if you're quick)."
```

Comment:
Maybe say `<N>` the current epoch if early in the epoch or `<N+1>` the next epoch if late in the epoch.